docs/VnVPlan/VnVPlan.tex

```diff
@@ -239,59 +231,654 @@ \section{System Tests}
 
 \subsection{Tests for Functional Requirements}
 
```

Comment:
Looks like this PR contains the Functional Requirements from the other PR, hopefully this does not result in conflicts and is up to date